client/me/security-checkup/account-recovery-phone.jsx

```diff
@@ -68,7 +68,7 @@ class SecurityCheckupAccountRecoveryPhone extends React.Component {
 				'You have set {{strong}}%(recoveryPhoneNumber)s{{/strong}} as your recovery SMS number.',
 				{
 					args: {
-						recoveryPhoneNumber: accountRecoveryPhone.numberFull,
+						recoveryPhoneNumber: '\u200e' + accountRecoveryPhone.numberFull,
 					},
 					components: {
 						strong: <strong />,
```

Comment:
Just a suggestion, what do you think about forcing the direction of the wrapping element, instead of using the ltr mark symbol?

```suggestion
						strong: <strong dir="ltr" />,
```